templates/articles/software_for_project_quality_management.json

```
@@ -0,0 +1,7 @@
+{
+  "title": "software_for_project_quality_management",

```

This field should contain the title of your blog post, but it seems you put the subpath there instead.
Could you replace it with something like:
```suggestion
  "title": "Software for Project Quality Management in Agile?",
```